docs/metrics_collection.md

```diff
@@ -0,0 +1,43 @@
+# Metrics collection guideline
+
+AsyncAPI **anonymously** tracks command executions to improve the specification and tools, ensuring no sensitive data reaches our servers. It aids in comprehending how AsyncAPI tools are used and adopted, facilitating ongoing improvements to our specifications and tools.
+
+Even though metrics collection is enabled by default, you can always [disable tracking](#how-to-disable-tracking) if you want to.
+
+## What we collect
+We are collecting the following metrics:
+
+- `asyncapi_adoption.action.invoked`:
+With this metric we are tracking the command executed on the CLI as soon as the command is invoked, so it has already been executed but not finished yet. We just want to know which commands are used, regardless they have failed or succeeded.
+
+Example of the data collected by this metric when the `validate` command has been executed:
+```
+asyncapi_adoption.action.invoked        COUNTER { action: 'validate' }  1
+```
+
+- `asyncapi_adoption.action.executed`:
```

Comment:
I remember we mentioned (before me going on holidays) the idea of renaming this one to `asyncapi_adoption.action.finished` or similar to be more clear about what is about.
We can merge this but please open a new PR whenever you can renaming it.

Reply:
Yeah, you are right! I agree with that renaming to avoid confusion 👍 Let me open the corresponding PR's